<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">#frame_4{
border-image:url(:/new/png/res/12.png);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>601</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;黑体&quot;;
color: rgb(255, 0, 127);</string>
       </property>
       <property name="text">
        <string>智能海洋工程与技术实验室上位机程序</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_zjupic">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>10</y>
         <width>210</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/new/png/res/13.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>选择保存文件地址</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>1190</x>
         <y>170</y>
         <width>211</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>    当前时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_time">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="portCombox">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>180</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_portComBOX">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>140</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>当前串口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start_recieve_btn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>230</y>
         <width>80</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>开始接收</string>
       </property>
      </widget>
      <widget class="QLabel" name="baud_label">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>140</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QComboBox" name="baud_rate_select">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_recieve_btn">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>230</y>
         <width>80</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>停止接收</string>
       </property>
      </widget>
      <widget class="Curve_chart" name="curve_chart7" native="true">
       <property name="geometry">
        <rect>
         <x>1170</x>
         <y>450</y>
         <width>339</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>270</y>
         <width>1111</width>
         <height>631</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="Curve_chart" name="curve_chart1" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Curve_chart" name="curve_chart2" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Curve_chart" name="curve_chart3" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="Curve_chart" name="curve_chart4" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Curve_chart" name="curve_chart5" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Curve_chart" name="curve_chart6" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Curve_chart</class>
   <extends>QWidget</extends>
   <header location="global">curve_chart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
